rskj-core/src/main/java/co/rsk/core/bc/BlockChainStatus.java

```diff
@@ -50,4 +51,8 @@ public BlockDifficulty getTotalDifficulty() {
     public boolean hasLowerTotalDifficultyThan(Status status) {
         return this.totalDifficulty.compareTo(status.getTotalDifficulty()) < 0;
     }
+
+    public boolean hasLowerDifficultyThan(LightStatus status) {
```

Comment:
Since the calling code that uses this method says "this works only for testing purposes", should this method have a similar comment? Or will it make sense later?

Reply:
That comment points to the "hasTotalDifficulty" variable. Not the whole method.